site/content/docs/reference/fips.md

```diff
@@ -0,0 +1,34 @@
+---
+title: FIPS-compatible builds of Pinniped binaries
+description: Reference for FIPS builds of Pinniped binaries
+cascade:
+  layout: docs
+menu:
+  docs:
+    name: FIPS-compatible builds of Pinniped binaries
+    weight: 30
+    parent: reference
+---
+By default, the Pinniped supervisor and concierge use ciphers that
+are not supported by FIPS 140-2. If you are deploying Pinniped in an
+environment with FIPS compliance requirements, you will have to build
+the binaries yourself using the `fips_strict` build tag and Golang's
+`go-boringcrypto` fork.
+
+The Pinniped team provides an [example Dockerfile](https://github.com/vmware-tanzu/pinniped/blob/main/hack/Dockerfile_fips)
+demonstrating how you can build Pinniped images in a FIPS compatible way.
+However, we do not provide official support for FIPS configuration, and we will not
+respond to GitHub issues opened related to FIPS support.
```

Comment:
@margocrawf Should we maybe remove the "and we will not respond to GitHub issues opened related to FIPS support" portion of this sentence? We might respond to certain kinds of FIPS questions or suggestions which align with our own desired usage of the FIPS example Dockerfile.

Reply:
Changed "will not" to "may not", do you think that makes it more clear?

Reply:
Thanks!